pkg/build/webhook/github/github_test.go

```diff
@@ -234,7 +234,7 @@ func setup(t *testing.T, filename, eventType string) *testContext {
 	}
 	req, err := http.NewRequest("POST", "http://origin.com", bytes.NewReader(event))
 	req.Header.Add("Content-Type", "application/json")
-	req.Header.Add("X-Github-Event", eventType)
+	req.Header.Add("X-GitHub-Event", eventType)
```

Comment:
i'm actually puzzed now, because our code is looking for X-GitHub-Event (even before this PR):
https://github.com/openshift/origin/blob/master/pkg/build/webhook/github/github.go#L101

so 1) i'm not sure how this test ever worked
2) not sure the code in webhook.go was ever correct but it sounds like it should be updated with this PR

maybe headers are case insensitive when doing header.Get()?